framework/include/systems/SystemBase.h

```diff
@@ -702,6 +733,12 @@ class SystemBase : public libMesh::ParallelObject
 
   /// Maximum number of dofs for any one variable on any one element
   size_t _max_var_n_dofs_per_elem;
+
+  /// Maximum number of dofs for any one variable on any one node
+  size_t _max_var_n_dofs_per_node;
+
+  /// Time integrator
+  std::shared_ptr<TimeIntegrator> _time_integrator;
```

Comment:
Why is this in SystemBase now? Seems specific to transient solves.

Reply:
`MooseVariables` in the displaced system need to be able to access the time integrator to compute the derivatives of `ad_u_dot`. Having a pointer to the `_time_integrator` in the `DisplacedSystem` seemed to be an easy way to accomplish that. But I'm open to suggestions